src/api/transaction.rs

```diff
@@ -432,7 +438,13 @@ fn operations_and_metadata_from_account_transaction_details(
                 to,
             ),
             Some(serde_json::to_value(&TransferredWithScheduleMetadata {
-                amounts: amount.clone(),
+                amounts: amount
+                    .iter()
+                    .map(|(t, a)| TimestampedAmount {
+                        timestamp:   *t,
+                        amount_uccd: *a,
+                    })
+                    .collect(),
```

Comment:
Then here you can do 
```
Vec::<TimestampedAmount>::from(amount)
```
or maybe
```
Vec::<TimestampedAmount>::from(&amount[..])
```

Reply:
I tried all kinds of combinations but am getting errors like
```
the trait `From<&Vec<(concordium_rust_sdk::crypto_common::types::Timestamp, concordium_rust_sdk::crypto_common::types::Amount)>>` is not implemented for `Vec<TimestampedAmount>`
```

Reply:
Ok. It's not important.